PyPy CI running no tests

Is that expected?

Example https://github.com/pandas-dev/pandas/runs/5593353924?check_suite_focus=true

```
2022-03-17T23:12:01.9910342Z   pythonLocation: /opt/hostedtoolcache/PyPy/3.8.12/x64/bin
2022-03-17T23:12:01.9910743Z ##[endgroup]
2022-03-17T23:12:02.1923011Z PYTHONHASHSEED=3717923424
2022-03-17T23:12:02.1941179Z xvfb-run pytest -r fEs -n auto --dist=loadfile --max-worker-restart 0 pandas -m "not slow and not network and not single_cpu"
2022-03-17T23:12:18.8783862Z ============================= test session starts ==============================
2022-03-17T23:12:18.8785081Z platform linux -- Python 3.8.12[pypy-7.3.8-final], pytest-7.1.1, pluggy-1.0.0
2022-03-17T23:12:18.8786140Z rootdir: /home/runner/work/pandas/pandas, configfile: pyproject.toml, testpaths: pandas
2022-03-17T23:12:18.8786880Z plugins: xdist-2.5.0, forked-1.4.0, hypothesis-6.39.4
2022-03-17T23:12:18.8787177Z gw0 I / gw1 I
2022-03-17T23:12:18.8787324Z 
2022-03-17T23:12:18.8787806Z ------ generated xml file: /home/runner/work/pandas/pandas/test-data.xml -------
2022-03-17T23:12:18.8788205Z ============================ no tests ran in 10.34s ============================
2022-03-17T23:12:18.8788536Z ERROR: Unknown config option: asyncio_mode
2022-03-17T23:12:18.8788748Z 
2022-03-17T23:12:18.8788755Z 
2022-03-17T23:12:18.9003569Z ##[error]Process completed with exit code 4.
```

--- a/.github/workflows/posix.yml
+++ b/.github/workflows/posix.yml
@@ -162,7 +162,7 @@ jobs:
       shell: bash
       run: |
         # TODO: re-enable cov, its slowing the tests down though
-        pip install Cython numpy python-dateutil pytz pytest>=6.0 pytest-xdist>=1.31.0 hypothesis>=5.5.3
+        pip install Cython numpy python-dateutil pytz pytest>=6.0 pytest-xdist>=1.31.0 pytest-asyncio hypothesis>=5.5.3
       if: ${{ env.IS_PYPY == 'true' }}
 
     - name: Build Pandas

